analyzers/src/SonarAnalyzer.CSharp/SymbolicExecution/Sonar/SonarExplodedGraph.cs

```diff
@@ -637,6 +637,10 @@ private ProgramState VisitParenthesizedVariableDesignationSyntax(ParenthesizedVa
         {
             foreach (var variable in GetVariableDesignationSyntaxCollection(designation))
             {
+                if (ParenthesizedVariableDesignationSyntaxWrapper.IsInstance(variable))
+                {
+                    continue;
+                }
```

Comment:
See also commit 8970ec4b2765f66990d465a238182fe369d3f184. I tried two approaches:

 * Recurse into VisitParenthesizedVariableDesignationSyntax
 * Skip ParenthesizedVariableDesignationSyntaxWrapper

Both approaches avoid the ArgumentNullExcpetion, don't show any difference in the tests (in both cases all tests are green and ITs don't show a difference) and the new FNs stay FNs.

If we go the "skip" approach I would turn it in a `Where(x => !ParenthesizedVariableDesignationSyntaxWrapper.IsInstance(x))`.